docs/user-guide/sdh/tenants.md

```diff
@@ -0,0 +1,78 @@
+# Tenants
+
+This section allows you to add new [tenants](#tenants) and map environments to these [installations](./README.md#installations) to ensure updates [rollout](#rollout) correctly. 
+
+## Adding Tenant
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to add tenants.
+{% endhint %}
+
+This involves the creation of new organizations where you wish to deploy s/w updates. Whenever you are onboarding a new client, you add them as a tenant. 
+
+1. Click **+ Add Tenant**.
+
+    ![Figure 1: Adding a Tenant](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/tenants-tab.jpg)
+
+2. Enter a name in **Tenant display name** field, e.g., `flareup.xyz`
+
+3. Add a unique identifier to your tenant in **Tenant ID** field, e.g., `flareup123` 
+
+4. (Optional) Add a description of the tenant.
+
+    ![Figure 2: Saving Tenant Details](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/add-tenant.jpg)
+
+5. Click **Save**.
+
+---
+
+## Adding Installation
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to add installations to a tenant.
+{% endhint %}
+
+This involves setting up [installation](./README.md#installations) for different environments, such as Prod, Development, and QA environments. You can consider these as licenses/installations your client has subscribed for.
+
+1. Click the tenant you created.
+
+    ![Figure 3: Clicking Your Tenant](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/tenant-list.jpg)
+
+2. Click **+ Add Installation**.
+
+    ![Figure 4: Adding an Installation](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/add-installation.jpg)
+
+3. Enter a name in **Installation display name** field, e.g., `Flareup Prod`
+
+4. Enter an **Installation ID**, e.g., `flareup-prod-1`
+
+    ![Figure 5: Entering Installation ID](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/installation-creation.jpg)
+
+5. Click **Save**.
+
+---
+
+## Mapping Environments
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to map environments.
+{% endhint %}
+
+This involves mapping customer's environments to the tenant installation so that your updates are deployed to the correct environments.
+
+1. Click **Map Environment**.
```

Comment:
I think we should provide a link to the doc page for "Creating an Environment" as well before talking about the mapping process.

Reply:
Will pick this up in the next iteration of this doc

Reply:
Thanks, added it now.